pkg/syncer/differ/decorations.go

```diff
@@ -47,7 +30,7 @@ func ManagementDisabled(obj client.Object) bool {
 // managed by Config Sync, because the annotation may be copied from another resource
 // managed by Config Sync.
 func ManagedByConfigSync(obj client.Object) bool {
-	return obj != nil && ManagementEnabled(obj) && core.GetAnnotation(obj, metadata.ResourceIDKey) == core.GKNN(obj)
+	return obj != nil && metadata.IsManagementEnabled(obj) && core.GetAnnotation(obj, metadata.ResourceIDKey) == core.GKNN(obj)
 }
 
 // ManagementUnset returns true if the resource has no Nomos ResourceManagementKey.
```

Comment:
Should this function also be moved to the `metadata` package?

Reply:
This one is kinda special, and probably misnamed. I wasn't sure what to call it if I moved it.

It doesn't just check the managed annotation. It also checks the resource-id annotation, which confirms that it was created by Config Sync and not just propagated to another resource by another controller.

Reply:
I was referring to the `ManagementUnset` function, looks like you've moved it in the latest change